robots/LoCoBot/install/install_gpmp2.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+# Install prerequisites
+sudo apt-get -y install python-numpy \
+						python-scipy \
+						python-matplotlib \
+						ipython \
+						ipython-notebook \
+						python-pandas \
+						python-sympy \
+						python-nose \
+						cmake \
+						libboost-all-dev
+pip install cython
+
+GPMP_FOLDER=~/gpmp_ws
+
+# Clone GTSAM and GPMP2
+mkdir GPMP_FOLDER
+cd GPMP_FOLDER
+git clone https://github.com/kalyanvasudev/gpmp2.git
```

Comment:
We might want to make a pull request to gpmp2 proper, make a release, and then have pyrobot depend on a specific version of gpmp2 and gtsam. This could be done when we bring gpmp2 up to the latest gtsam release so everything is kosher.